gpuallocator/allocator.go

```diff
@@ -6,7 +6,7 @@ import (
 	"fmt"
 	"runtime"
 
-	"github.com/NVIDIA/gpu-monitoring-tools/bindings/go/nvml"
```

Comment:
Can we merge [this](https://github.com/NVIDIA/go-nvlib/pull/6) and use the new github.com import path?

Reply:
That PR has been merged and this PR updated accordingly.